app/workers/import_profile.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Workers
+  class ImportProfile < Base
+    sidekiq_options queue: :medium
+
+    include Diaspora::Logging
+
+    def perform(user_id)
+      user = User.find_by(username: user_id)
```

Comment:
Changed to user.id - is the check really unneeded?

Reply:
If the user existed when the job was created, it will exist when it is executed, since diaspora never deletes users. So I don't see why the check would be needed. The only reason this could fail would be if a podmin manually does things in the database, and when podmins do that, all sort of things break, and then it can just fail hard.